webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -1,153 +1,98 @@
 import PropTypes from 'prop-types';
-import React, { useState } from 'react';
-import Skeleton from 'react-loading-skeleton';
-import { ArrowIcon } from '@patternfly/react-icons';
-import ElipsisWithTooltip from 'react-ellipsis-with-tooltip';
+import React from 'react';
 import {
-  Card,
+  Bullseye,
   DataList,
+  Card,
+  CardActions,
+  CardHeader,
+  CardTitle,
+  CardBody,
   DataListItem,
   DataListItemRow,
   DataListItemCells,
+  DataListWrapModifier,
   DataListCell,
-  CardBody,
-  CardTitle,
-  Accordion,
-  AccordionItem,
-  AccordionContent,
-  AccordionToggle,
+  Title,
 } from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
+import URI from 'urijs';
 
-import { foremanUrl } from '../../../../foreman_tools';
+import { foremanUrl } from '../../../common/helpers';
 import { translate as __ } from '../../../common/I18n';
 import { useAPI } from '../../../common/hooks/API/APIHooks';
+import RelativeDateTime from '../../common/dates/RelativeDateTime';
+import SkeletonLoader from '../../common/SkeletonLoader';
+import { STATUS } from '../../../constants';
 
-const AuditCard = ({ hostName }) => {
-  const [activeAccordion, setActiveAccordion] = useState(0);
-  const onToggle = id => {
-    if (id === activeAccordion) {
-      setActiveAccordion('');
-    } else {
-      setActiveAccordion(id);
-    }
-  };
+const NUMBER_OF_RECORDS = 3;
+const BASE_URL = '/audits';
 
-  const url = hostName && foremanUrl(`/api/audits?search=host+%3D+${hostName}`);
+const AuditCard = ({ hostSearch }) => {
+  const host = `host=${hostSearch}`;
```

Comment:
I think I didn't explain myself well, what I meant was
```suggestion
const AuditCard = ({ hostName }) => {
  const hostSearch = `host=${hostName}`;
```

Reply:
Makes much more sense :D